misc/tremulous-grangerhub.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Type=Application
+Name=GrangerHub Tremulous
+Comment=A free team based FPS/RTS hybrid built on the ioq3 engine. Includes GrangerHub community updates.
+Icon=/usr/share/pixmaps/tremulous.xpm
```

Comment:
This is wrong.

1. The icon should not have a hardcoded path. It should ideally be just "tremulous", "tremulous-grangerhub" or "io.github.grangerhub.Tremulous".
2. The icon format should be png or svg.
3. The icon has to be installed in a standard location: "$PREFIX/share/icons/hicolor/*size*/apps" (replace *size* with the actual size, e.g. 512x512) or "/usr/share/icons/hicolor/scalable/apps" for a svg one.

Reply:
Also, again - why not using the desktop file from my Flatpak?

Reply:
By the way (but this is not a requirement), the icon name should ideally be the same as the desktop file name.